integration_test/third_party_apps_data/applications/active_directory_ds/metadata.yaml

```diff
@@ -1,5 +1,6 @@
 short_name: AD DS
 long_name: Active Directory Domain Services
+supported_app_version: ["windows-server-2012-r2", "windows-server-2016", "windows-server-2019", "windows-server-2022"]
```

Comment:
While 2022 was included in the scoping doc, we haven't officially added support for 2022 to Ops Agent yet, so we should probably remove that from the list for now. We can add it back in once we add 2022 support.